js_modules/dagster-ui/packages/ui-core/src/runs/HourlyDataCache/HourlyDataCache.tsx

```diff
@@ -17,7 +17,7 @@ export class HourlyDataCache<T> {
   constructor(id?: string | false) {
     if (id) {
       this.indexedDBCache = cache<'hourlyData', typeof this.cache>({
-        dbName: `HourlyDataCache:${id}`,
+        dbName: `HourlyDataCache2:${id}`,
```

Comment:
Changing the DB name here because I think the old one will have duplicate runs due to this issue. I need to setup a way to clean up the old ones when change names like this.

Reply:
on second thought I guess we can live with the duplication and wait for the cached entries to expire in a week

Reply:
will we clean up the old entries if we change the name?

Reply:
not atm no

Reply:
based on async conversation i think the move is to just leave the key as is